distribution/docs/src/main/resources/_contents/_endpoints/csw-endpoint-contents.adoc

```diff
@@ -958,9 +958,14 @@ Namespace:: In POST operations, namespaces are defined in the XML. In GET operat
 Service:: The service being used, in this case it is fixed at CSW.
 Version:: The version of the service being used (2.0.2).
 OutputFormat:: The requester wants the response to be in this intended output. Currently, only one format is supported (application/xml). If this parameter is supplied, it is validated against the known type. If this parameter is not supported, it passes through and returns the XML response upon success.
-OutputSchema:: This is the schema language from the request. This is validated against the known list of schema languages supported (refer to http://www.w3.org/XML/Schema).
+OutputSchema:: The OutputSchema indicates which schema shall be used to generate the response to the GetRecords operation.  The supported output schemas are listed in the GetCapabilities response.
```

Comment:
@millerw8 - I think this is technically controlled by the TypeName attribute in the GetRecords request.  For example `typeNames="Record"` means the query schema is Dublin Core Record.  So technically you could query by a one schema type and get results in a different one using the `outputSchema` parameter.